src/EditorFeatures/VisualBasicTest/ExtractMethod/ExtractMethodTests.ControlFlowAnalysis.vb

```diff
@@ -282,7 +282,7 @@ End Class</text>
             End Function
 
             <WorkItem(540154, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/540154")>
-            <Fact, Trait(Traits.Feature, Traits.Features.ExtractMethod)>
+            <Fact(Skip:= "https://github.com/dotnet/roslyn/issues/15223"), Trait(Traits.Feature, Traits.Features.ExtractMethod)>
```

Comment:
Based on the test result and issue #15223, it should be `Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.ExtractMethod.ExtractMethodTests+FlowAnalysis.BugFix6313_6` instead.